target/linux/realtek/dts-5.10/rtl8382_d-link_dgs-1210-16.dts

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: GPL-2.0-or-later OR MIT
 
-#include "rtl8382_d-link_dgs-1210.dtsi"
+#include "rtl838x_d-link_dgs-1210_soc.dtsi"
 
```

Comment:
Any reason you did not include the gpio related dtsi here as well?

Reply:
Yes. The answer is very much the same as your -10p question. This is how the hardware in the -16.dts file was defined and I don't do changes on the hardware in this project. I have only made a 1:1 rebuild so was it there before it will be there after.